tests/SqlFormatterTest.php

```diff
@@ -99,13 +101,22 @@ public function testUsePre(): void
         $this->assertSame($actual, $expected);
     }
 
+    /** @return string[] */
+    private static function fileDataSql(): array
```

Comment:
`fileDataProvider` is a PHPUnit data provider… or close to it. But `fileDataSql` is not a "data sql", that's not a thing. I'd use `sqlData`, or `fileSqlData`, or `dataFromSqlFile()`

Reply:
Renamed this and every related and added providers.